flocker/node/script.py

```diff
@@ -174,6 +174,7 @@ def flocker_dataset_agent_main():
     # Later, construction of this object can be moved into
     # DatasetAgentServiceFactory.get_service where various options passed on
     # the command line could alter what is created and how it is initialized.
+    # XXX: Implement the factory proposed above (here or in a blocking / follow up issue)
```

Comment:
(And then I start thinking that centralizing the config at the control service is actually the easier path...)